src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -45,12 +45,6 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 		finished?()
 	}
 
-	// MARK: - Protocol DismissHandling
```

Comment:
Question, if we don't implement the Protocol do we still need to conform to it?

Reply:
yes, if we conform to it the default implementation (do not dismiss) will get called